solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -151,27 +152,42 @@ contract RandomBeacon is IRandomBeacon, Ownable {
     ///         operator affected.
     uint256 public dkgMaliciousResultNotificationRewardMultiplier;
 
+    /// @notice Calculated gas cost for submitting a DKG result. This will
+    ///         be refunded as part of the DKG approval process. It is in the
+    ///         submitter's interest to not skip his priority turn on the approval,
+    ///         otherwise the refund of the DKG submission will be refunded to
+    ///         other member that will call the DKG approve function.
+    uint256 public dkgResultSubmissionGas = 235000;
+
+    // @notice Gas is meant to balance the DKG approval's overall cost. Can be
+    //         updated by the governace based on the current market conditions.
+    uint256 public dkgApprovalGasOffset = 43500;
+
+    // @notice Gas is meant to balance the failed heartbeat cost. Can be updated
+    //         by the governace based on the current market conditions.
+    uint256 public failedHeartbeatGasOffset = 54500;
+
+    // @notice Gas is meant to balance the relay entry submission cost. Can be
+    //         updated by the governace based on the current market conditions.
+    uint256 public relayEntrySubmissionGasOffset = 11500;
+
     // Other parameters
 
     /// @notice Stores current operator inactivity claim nonce for given group.
     ///         Each claim is made with an unique nonce which protects
     ///         against claim replay.
     mapping(uint64 => uint256) public inactivityClaimNonce; // groupId -> nonce
 
+    /// @notice Authorized contracts that can request a relay entry without
+    ///         sending any fees.
+    mapping(address => bool) public authorizedContracts;
```

Comment:
So maybe we should rename it to `authorizedRequesters`? This will be more explicit about what they are authorized for.

Reply:
Done [dd9ad55](https://github.com/keep-network/keep-core/pull/2860/commits/dd9ad55739999e3d5bdc88bdc68b84b1c20367f9)